azure-pipeline-templates/verbose-tests.yml

```diff
@@ -123,7 +123,7 @@ steps:
         adls: ${{ parameters.adls }}
         idstring: ${{ parameters.service }} with Key Credentials
         distro_name: ${{ parameters.distro_name }}
-        clone: "true"
+        # clone: true
```

Comment:
why keep this commented? Why not remove it?

Reply:
will cleanup.